app/templates/gulp/tasks/server.coffee

```diff
@@ -1,7 +1,8 @@
-serverFile = path.join(config.BASE_LIB_PATH, 'webserver')
+nodemon = require("gulp-nodemon")
 
-coffee = require('coffee-script/register')
-server = require(serverFile)
+cfg = require(path.join(config.ROOT_PATH, 'nodemon.json'))
```

Comment:
It used to say nodemonConfig... and I got sick of typing it while trying to figure out how to exploit it... I can change it to nodemonConfig (config is taken)